rcl/test/rcl/test_get_actual_qos.cpp

```diff
@@ -42,29 +42,32 @@
     INSTANTIATE_TEST_CASE_P, instance_name, CLASSNAME(test_case_name, \
     RMW_IMPLEMENTATION), __VA_ARGS__)
 
+// One const for each of the RMW_IMPLEMENTATION values
+const char * rmw_fastrtps_cpp = "rmw_fastrtps_cpp";
+const char * rmw_fastrtps_dynamic_cpp = "rmw_fastrtps_dynamic_cpp";
```

Comment:
Are these used?

Reply:
RMW_IMPLEMENTATION is one of the symbols rmw_fastrtps_cpp, rmw_fastrtps_dynamic_cpp, etc, depending on the underlying rmw implementation used (I will add the others when ending with each implementation).
Maybe it is not too clear, but I didn't have a better idea.

Reply:
To be clearer, it's used here:
https://github.com/ros2/rcl/blob/cbd46cbe69365b5a60de4f86312aebd3d3cacf59/rcl/test/rcl/test_get_actual_qos.cpp#L185-L187

Reply:
How are they used there? I see you're creating an `std::string` using a literal of the same name, but I don't see the `const char *` declared at the top of the file being used here...

Reply:
RMW_IMPLEMENTATION was a token, not an string, which was exactly one of the const on the top of the file. A really bad idea. I solved this in a proper way now.